### PR TITLE
Added guide on How to post to social media

### PR DESCRIPTION
**Pull request by AlbaRoza**
New guide on How to post to social media added. Includes tips on how to use Planable as a scheduling tool for tweets and posts.

-----
[View rendered activities/communication/how-to-post-to-social-media.md](https://github.com/publiccodenet/about/blob/AlbaRoza/activities/communication/how-to-post-to-social-media.md)
[View rendered activities/communication/index.md](https://github.com/publiccodenet/about/blob/AlbaRoza/activities/communication/index.md)

### CODE REVIEW

**Comment by ElenaFdR** [comment]
@alba Travis is blocking this due to incorrect Markdown. Shall we work on that together?

File: activities/communication/how-to-post-to-social-media.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Use the right tone in your post. Notice that enthusiasm varies whether we're announcing something that excites us or not.
+* Tag the organizations or people you are writing about if possible.
+* Use relevant hashtags like #PublicCode,  #StandardforPublicCode, #ltapc, #OpenSource, #FOSS, #GovTech
+* In order to make the message more engaging, include an image and/or video with every post. Note that links usually unfold a preview of the content. Emojis are also welcome in moderation.
+* Always include a link for further information in case people are intrigued by what you've written.
+
+## Process
+
+* Make sure you have access to Planable. If not, ask the [communications team](https://about.publiccode.net/organization/staff.html#communications).
+* Select the social media channel/s where you want your message to be published. Note that LinkedIn's integration doesn't work great when mentioning a person or company (@____).
```
Comment (felixfaassen) [inline activities/communication/how-to-post-to-social-media.md:45]: Does this mean we should not use the integration or check if the @ mentions in LinkedIn work correctly

**Review by felixfaassen — APPROVED** [review]
I have no blocking remarks. I would like to do it myself once in order to see if the process description helps with doing the actual work. But I will soon find out. I added a small comment that it could help to clarify the " linkedin integration" not working.

File: activities/communication/how-to-post-to-social-media.md (hunk 1 above)
Comment (ElenaFdR) [reply activities/communication/how-to-post-to-social-media.md:45]: It doesn't really work, so our workaround has been to edit LI posts as soon as they're live to correctly tag people. Would it be more helpful to include this instruction explicitly?

File: activities/communication/how-to-post-to-social-media.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Our goals for Planable are:
+
+* to make it easy for staff to create social posts about what's interesting or exciting in their work
+* to give the communications team enough time to review social media posts without it being stressful
+* to give everyone a structured and predictable social media post creation and publishing process
+
+## Why we post to social media
+
+We use social media to share information that's interesting or exciting to us. Our information should be timely, and it's ok if posts are no longer be relevant a week after they're published.
```
Comment (Ainali) [inline activities/communication/how-to-post-to-social-media.md:18]:
```suggestion
We use social media to share information that's interesting or exciting to us. Our information should be timely, and it's ok if posts are no longer relevant a week after they're published.
```

File: activities/communication/how-to-post-to-social-media.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+[Read the commmunications plan](https://about.publiccode.net/activities/communication/communications-plan.html) for more information about our audiences and messages.
+
+## Tips for writing social media posts
+
+* The most interesting posts are the ones from our community and codebases. Try to showcase their success stories before ours.
+* Make sure the audience of your post matches the Foundation for Public Code one.
+* Keep in mind audiences in other time zones. Is your message important for them to read?
+* Use the right tone in your post. Notice that enthusiasm varies whether we're announcing something that excites us or not.
+* Tag the organizations or people you are writing about if possible.
+* Use relevant hashtags like #PublicCode,  #StandardforPublicCode, #ltapc, #OpenSource, #FOSS, #GovTech
```
Comment (Ainali) [inline activities/communication/how-to-post-to-social-media.md:38]:
```suggestion
* Use relevant hashtags like #PublicCode, #StandardforPublicCode, #ltapc, #OpenSource, #FOSS, #GovTech
```

File: activities/communication/how-to-post-to-social-media.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* Always include a link for further information in case people are intrigued by what you've written.
+
+## Process
+
+* Make sure you have access to Planable. If not, ask the [communications team](https://about.publiccode.net/organization/staff.html#communications).
+* Select the social media channel/s where you want your message to be published. Note that LinkedIn's integration doesn't work great when mentioning a person or company (@____).
+* Post your draft in Planable timeline. Note that this may not be the final message since it's subject to review.
+* Add a tag with your name so the communications team can reach out to you with questions.
+* Schedule the message at least 24 hours in advance so the communications team have a margin to review the message.
+* Check if there are already other scheduled posts for your preferred publishing time.
```
Comment (Ainali) [inline activities/communication/how-to-post-to-social-media.md:49]:
```suggestion
* Check if there are already other scheduled posts for your preferred publishing time. If there are, start a conversation with the authors of those to finalize the scheduling.
```

**Review by Ainali — CHANGES_REQUESTED** [review]
Wonderful and valuable guidance!

I have suggested a few tiny tweaks, and then it's good to go.

File: activities/communication/how-to-post-to-social-media.md (hunk 1 above)
Comment (ElenaFdR) [reply activities/communication/how-to-post-to-social-media.md]: I've updated this with more explicit instructions.

**Review by ElenaFdR — APPROVED** [review]

**Review by Ainali — APPROVED** [review]

**Comment by AlbaRoza** [comment]
Ready to go! 🚀

**Pull request merged**